manifests/init.pp

```diff
@@ -63,6 +63,7 @@
   $watches           = {},
   $checks            = {},
   $acls              = {},
+  $umask             = '0022',
 ) inherits consul::params {
 
```

Comment:
can you validate_re on this?

Also it would be a double standard for me to insist on tests from external people and not from contributors?
I would mostly want to be sure you got the name of the variable right (we don't run tests with strict variables, we probably should) also that each init script has the correct syntax for umask maybe?